src/mono/wasm/runtime/pthreads/worker/index.ts

```diff
@@ -114,7 +114,7 @@ function onMonoConfigReceived(config: MonoConfigInternal): void {
 export function mono_wasm_pthread_on_pthread_attached(pthread_id: pthread_ptr): void {
     const self = pthread_self;
     mono_assert(self !== null && self.pthread_id == pthread_id, "expected pthread_self to be set already when attaching");
-    console.debug("MONO_WASM: attaching pthread to runtime", pthread_id);
+    console.debug("MONO_WASM: attaching pthread to runtime 0x" + pthread_id.toString(16));
```

Comment:
Please add `if (runtimeHelpers.diagnosticTracing) ` ...
Ideally all places which are missing it.